src/www/app/outline_server_repository/access_key_serialization.ts

```diff
@@ -36,8 +36,16 @@ export function staticKeyToShadowsocksSessionConfig(staticKey: string): Shadowso
   }
 }
 
-function parseShadowsocksSessionConfigJson(maybeJsonText: string): ShadowsocksSessionConfig | null {
-  const {method, password, server, server_port, prefix} = JSON.parse(maybeJsonText);
+function parseShadowsocksSessionConfigJson(
+  maybeJsonText: string
+): ShadowsocksSessionConfig | errors.SessionConfigError | null {
+  const responseJson = JSON.parse(maybeJsonText);
```

Comment:
"maybe" is usually a bad name. It doesn't say what it is.
You can name it `responseText` or `responseBody`.